core/src/main/java/bisq/core/btc/wallet/http/MemPoolSpaceTxBroadcaster.java

```diff
@@ -115,8 +114,21 @@ public void onSuccess(String txId) {
             }
 
             public void onFailure(@NotNull Throwable throwable) {
-                log.warn("Broadcast of raw tx with txId {} to {} failed. Error={}",
-                        txIdToSend, url, throwable.toString());
+                Throwable cause = throwable.getCause();
+                if (cause instanceof HttpException) {
+                    int responseCode = ((HttpException) cause).getResponseCode();
+                    String message = cause.getMessage();
+                    if (responseCode == 400 && message.equals("sendrawtransaction RPC error: {\"code\":-27,\"message\":\"Transaction already in block chain\"}")) {
```

Comment:
Instead of checking for that exact message, maybe just checking for a substring of `\"code\":-27` is better, since the exact string might change in future releases of Bitcoin Core RPC

Reply:
ok. do you have a list of other return codes/messages?

Reply:
https://github.com/bitcoin/bitcoin/blob/master/src/rpc/protocol.h